test/TestProjectWithNoNuspecForPacking/TestProjectWithNoNuspecForPacking.csproj

```diff
@@ -0,0 +1,24 @@
+<Project>
+  <Import Project="$(MSBuildThisFileDirectory)..\..\src\MSBuild.Sdk.SqlProj\Sdk\Sdk.props" />
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <SqlServerVersion>Sql150</SqlServerVersion>
+    <RecoveryMode>Simple</RecoveryMode>
+    <AllowSnapshotIsolation>True</AllowSnapshotIsolation>
+    <ReadCommittedSnapshot>True</ReadCommittedSnapshot>
+    <ServiceBrokerOption>EnableBroker</ServiceBrokerOption>
+    <PackageProjectUrl>https://github.com/rr-wfm/MSBuild.Sdk.SqlProj/</PackageProjectUrl>
+    <Authors>MSBuild.SDK.SqlProj team</Authors>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <ExcludeObjectTypes>Users,Assemblies</ExcludeObjectTypes>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Include="$(OutputPath)\*.dacpac" Pack="true" PackagePath="tools/" />
```

Comment:
Wonder if this could be included in the Sdk.props ?

Reply:
Good question, isn't it already?

Reply:
No, or at least I cannot see it

Reply:
If I remove the ItemGroup, I get this error:

`error NU5017: Cannot create a package that has no dependencies nor content.`

Reply:
That would mean that running a `dotnet pack` on a newly created project would result in an empty NuGet package right?

Reply:
Then I think we should move this into the `Sdk.props` file so that it works out of the box.

Reply:
The built Nuget package would contain a .dacpac without any content, I guess

Reply:
I have move it to `Sdk.Props`